crates/modelardb_manager/src/remote.rs

```diff
@@ -460,7 +460,11 @@ impl FlightService for FlightServiceHandler {
         };
 
         // Confirm the SQL statement was executed.
-        Ok(Response::new(Box::pin(stream::empty())))
+        let options = IpcWriteOptions::default();
```

Comment:
Unclear why [`stream::empty()` ](https://docs.rs/futures/latest/futures/prelude/stream/fn.empty.html) was replaced with code that also seems to create an empty stream. Maybe extend the comment to state what the stream returns, and if it is empty, why `stream::empty()` is not used.

Reply:
The new code does not create an empty stream, it just creates a stream of a single element. It seems that a schema is required for `do_get` to execute correctly in PyArrow and possibly also other clients. I have made it clear in the comment that the stream is not empty.